conference.js

```diff
@@ -1359,14 +1359,17 @@ export default {
 
     /**
      * Toggles between screensharing and camera video.
+     *
+     * @param {boolean} toggle - Indicates whether screen sharing should be
```

Comment:
have you tested what will happen if you call this method twice either with true or false ? will it crash or is it handled gracefully ?

What if the screen sharing is turned on already ? I suppose it will ask for new stream (but I haven't tested it). Is that ok ? If yes then it should be described in the doc how those cases are handled exactly.